doc/source/template_docker_nextflow.config

```diff
@@ -8,7 +8,11 @@ docker {
 }
 
 // CLUSTER SETTINGS:
-executor.name = ''  // set cluster type, e.g. sge
+executor {
+    name = ''                   // e.g. sge, local
+    // queueSize =              // e.g. 4, to limit the number of processes simultaneously handled
+    // cpus =                   // e.g. 8, to limit the CPU resources when using local executor
```

Comment:
```suggestion
    // cpus =                   // e.g. 3, to limit the CPU resources when using the 'local' executor
```

Reply:
8 seemed like a high number to use as an example limit for a local computer, not many laptops have more than 8 CPUs/cores.